models/articles.model.js

```diff
@@ -31,6 +31,19 @@ exports.selectArticleById = (article_id) => {
 		});
 };
 
-exports.selectAllCommentsForArticle = () => {
-	return;
+exports.selectAllCommentsForArticle = (article_id) => {
+	return db
+		.query(
+			"SELECT * FROM comments WHERE comments.article_id = $1 ORDER BY created_at DESC;",
+			[article_id]
```

Comment:
spot on SQL statement, great to see you've ordered and sorted the returned data.